extensions/geode-modules-tomcat8/src/distributedTest/java/org/apache/geode/modules/session/TestSessionsTomcat8Base.java

```diff
@@ -399,7 +407,7 @@ public void testExtraSessionsNotCreated() throws Exception {
 
     WebResponse response = wc.getResponse(req);
     assertThat(response.getText()).isEqualTo("done");
-    assertThat(region.size()).as("The region should be empty").isEqualTo(0);
+    assertThat(region.size()).as("The region should be empty").isEqualTo(1);
```

Comment:
Would you either update the description or make it so that we're really testing for an empty region?